.travis.yml

```diff
@@ -4,9 +4,10 @@ language: node_js
 addons:
   chrome: stable
 node_js:
-  - "7"
-  - "node"
+  - "8.9"
+  - "8.9"
```

Comment:
Is this required for Angular CLI tools?

Reply:
Yes, Angular CLI only works with NodeJS 8.9 and above